packages/sdk-cli/src/models/localRelay/launch.ts

```diff
@@ -25,5 +24,7 @@ export function launch(
   // unref()'ing child process allows the parent process (CLI) to exit without waiting for the child (Local Relay) to exit.
   relayChildProcess.unref();
 
-  return relayChildProcess;
+  return new Promise<ChildProcess>((resolve) =>
```

Comment:
To be safe, add a rejection on `error`.

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/680048b82c1fae3414b669d2fd23a4ced3669690

Reply:
https://github.com/Fitbit/developer-bridge/pull/176/commits/680048b82c1fae3414b669d2fd23a4ced3669690